### ISSUE TITLE
Cannot search partners if dropdown opens upwards

### ISSUE THREAD

**Issue by ntrncic** [unicef/etools-issues#570]
Unable to see the search field when dropdown open upwards

**Comment by ntrncic** [comment]
![Screen Shot 2017-08-03 at 10.41.01 AM.png](https://waffleio-direct-uploads-production.s3.amazonaws.com/uploads/58c83b76afe7881a002d4647/125516c66e82c728ace21e0d46db978826878de9bce1b057e40fa0c57e0b772b6454f06b7191a379ef0c2f50415f4fbc30254d58eeb9cb74fce36f34985619e8d93f5870c76917aeec874cb809ee218b4ed7beeadb4fc1fa2137a5dad11b61435c52dcbd4f3fb29a4ac259f34be04a52a5103e61e673376e0a.png)

**Comment by robertavram** [comment]
does this relate to #405 ?

**Comment by acory** [comment]
@robertavram  Yes, same issue,  just the droprown for which it reproduces is different

**Comment by roxanavlad** [comment]
test result: pass
tested on dev environment on google chrome and internet explorer

**Comment by mpawlowski85** [comment]
cannot reproduce the issue. 
@ntrncic please close it if this is no longer an issue.

**Comment by mpawlowski85** [comment]
cannot reproduce this any longer - moving to QA passed.
